web/templates/settings/clansettings.html

```diff
@@ -21,10 +21,11 @@
               {{ $desc := or $d.description.String "" }}
               {{ $icon := or $d.icon.String "" }}
               {{ $bg := or $d.background.String "" }}
+              {{ $status := or $d.status.String "" }}
```

Comment:
```suggestion
              {{ $status := or $d.status.Int 0 }}
```

Reply:
untested but it's an int on api side https://github.com/osuAkatsuki/akatsuki-api/blob/master/app/v1/clan.go#L383